pkg/apis/discovery/install/install.go

```diff
@@ -34,7 +34,7 @@ func init() {
 // Install registers the API group and adds types to a scheme
 func Install(scheme *runtime.Scheme) {
 	utilruntime.Must(discovery.AddToScheme(scheme))
-	utilruntime.Must(v1alpha1.AddToScheme(scheme))
 	utilruntime.Must(v1beta1.AddToScheme(scheme))
-	utilruntime.Must(scheme.SetVersionPriority(v1beta1.SchemeGroupVersion, v1alpha1.SchemeGroupVersion))
+	utilruntime.Must(v1.AddToScheme(scheme))
+	utilruntime.Must(scheme.SetVersionPriority(v1.SchemeGroupVersion, v1beta1.SchemeGroupVersion))
```

Comment:
prefer v1beta1 over v1 until 1.22, so that etcd storage writes using v1beta1, so that 1.20 API servers can read the objects we write.

Reply:
good catch

Reply:
Fixed so etcd storage writes to v1beta1.